app/bundles/IntegrationsBundle/Controller/ConfigController.php

```diff
@@ -152,7 +152,10 @@ private function submitForm()
         }
 
         // Show the form if there are errors
-        if (!$this->form->isValid()) {
+        if (!$this->form->isValid() && (!$this->integrationObject instanceof ConfigFormAuthorizeButtonInterface || $this->integrationObject->isAuthorized())) {
```

Comment:
This will fail for the following scenario,
When the plugin is configured the first time and the user tries to save the configuration without entering values in the required fields. It will not throw the NotBlank constraint even if `!$this->form->isValid()` is true. As `isAuthorized` will always send false for fresh configuration.

Reply:
I can't reproduce with empty plugin configuration
<img width="570" alt="Snímek obrazovky 2020-11-20 v 15 12 00" src="https://user-images.githubusercontent.com/12815758/99810891-acb14000-2b44-11eb-97aa-99433d909dc2.png">

Are you able to record video or describe which plugin was used and write exact steps?
TY!